tests/test_autodoc.py

```diff
@@ -123,7 +123,7 @@ def process_docstring(app, what, name, obj, options, lines):
         lines.extend(['42', ''])
 
 
-def process_signature(app, what, name, obj, options, args, retann):
+def process_signature(app, what, name, obj, options, args, retann, documenter):
```

Comment:
If you undo this the warning will show in `tox -e py`.  Is there a desire to have a test that uses the 7 parameter version and maybe just clear the warnings?